### DISCUSSION TITLE
[UserStoriesToRefine] #3 As <user> I want a website where UBC's videos can be easily accessed and watched

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#23] [General]
### As <user> I want a website where UBC's videos can be easily accessed and watched

**Comment by nhrafiq** [comment]
# As a cacao producer, I want to use the UBC website to access and watch videos so that I can remain up to date with the chocolate industry. 

## Level 2: As a cacao producer I want to easily locate the video library, filter / search the library, and browse the library so that I can find videos that are both informative and relevant to my personal needs. 
Acceptance criteria: 
* It should be easy to navigate to the video library page from any page on UBC's website (such as a button that is clearly identifiable as the link to the library)
* User stories 1's criteria are relevant for the video locating functionality: https://github.com/emecas/civic-tech-website-consultancy-pilot-translation/discussions/20 
## Level 2: As a cacao producer I want to watch videos so that I can gain necessary information and updates from them. 
Acceptance criteria:
* Video playback should be intuitive (providing functions like play, skip forward/backward, pause, and captions(?))
* A video's content should be easy to understand by viewing the tags/labels, title, and description so users can understand what information is communicated within the video before watching it.
